#! /bin/sh -e
## 50norwegian_chart.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Correct the norwegian general chart.
## DP: Sent upstream 2004-11-27.

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch)   ( cd BUILD/sql-ledger && patch $patch_opts -p1 ) < $0 ;;
       -unpatch) ( cd BUILD/sql-ledger && patch $patch_opts -p1 -R ) < $0 ;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0
@DPATCH@
--- sql-ledger-2.4.5/sql/Norwegian_Default-chart.sql	Wed Aug  4 22:21:26 2004
+++ sql-ledger/sql/Norwegian_Default-chart.sql	Sat Nov 27 22:35:26 2004
@@ -12,6 +12,7 @@
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1070', 'Utsatt skattefordel', 'A','', 'A', '');
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1080', 'Goodwill', 'A','', 'A', '');
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('11', 'Varige driftsmidler', 'H', '', 'A', '');
+insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1080', 'Goodwill', 'A','', 'A', '');
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1100', 'Bygninger', 'A','', 'A', '');
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1120', 'Bygningsmessige anlegg', 'A','', 'A', '');
 insert into chart (accno,description,charttype,gifi_accno,category,link) values ('1140', 'Jord- og skogbrukseiendommer', 'A','', 'A', '');
